custom_components/birthdays/__init__.py

```diff
@@ -115,12 +137,16 @@ async def update_data(self, *_):
 
         days_until_next_birthday = (next_birthday-today).days
 
-        self._age_at_next_birthday = next_birthday.year - self._date_of_birth.year
+        age = next_birthday.year - self._date_of_birth.year
+        self._age_at_next_birthday = age
```

Comment:
With line 75 gone, line 141 can also be removed (we store age only in extra_attributes)

Reply:
Removed!